examples/opentelemetry-tracing/docker-compose.yaml

```diff
@@ -0,0 +1,59 @@
+version: "3.8"
+services:
+
+  # base images
+  flask_service:
```

Comment:
i think this is probably necessary - as the final destination for any requests, but i dont see it used